### PR TITLE
added company get and update documentation

### PR DESCRIPTION
**Pull request by mariusmel**

### CODE REVIEW

File: sections/company.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+| chargebee_id    | Integer                     |
+| city            | String                      |
+| contact_email   | String                      |
+| contact_name    | String                      |
+| contact_phone   | String                      |
+| country         | String (ISO 3166-1 Alpha 2) |
+| currency        | String                      |
+| company_name    | String                      |
+| street          | String                      |
+| company_name    | String                      |
```
Comment (tilds) [inline sections/company.md:19]: duplicate company_name field

File: sections/company.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+| -------------- | --------------------------- |
+| city           | String                      |
+| contact_email  | String                      |
+| contact_name   | String                      |
+| contact_phone  | String                      |
+| country        | String (ISO 3166-1 Alpha 2) |
+| currency       | String                      |
+| company_name   | String                      |
+| street         | String                      |
+| company_name   | String                      |
```
Comment (tilds) [inline sections/company.md:63]: duplicate company_name field

**Review by tilds — CHANGES_REQUESTED** [review]
We should remove chargebee_id and company_id from the response

**Comment by danny-larsen** [comment]
I don't think we should expose the company id and the chargebee id

**Comment by danny-larsen** [comment]
Maybe also seats and tier?

File: sections/company.md (hunk 1 above)
Comment (mariusmel) [reply sections/company.md:19]: removed it

File: sections/persons.md [hunk 3]
```diff
@@ -26,6 +26,8 @@
 | default_role        | Integer, ID of default role                                                            |
 | cost                | Decimal, cost from the current cost period                                             |
 | language            | String {"SPANISH", "DANISH", "FRENCH", "ENGLISH_EU", "ENGLISH_UK", "ENGLISH_US"}       |
+| start_date          | String                                                                                 |
+| end_date            | String                                                                                 |
```
Comment (tilds) [inline sections/persons.md:30]: Can you put "ISO 8601" in the Description/format column here?

**Review by tilds — CHANGES_REQUESTED** [review]

File: sections/persons.md (hunk 3 above)
Comment (mariusmel) [reply sections/persons.md]: added it

**Pull request merged**